src/pages/projects/form/ProjectParticipants.tsx

```diff
@@ -0,0 +1,99 @@
+import AddIcon from '@mui/icons-material/AddCircleOutlineSharp';
+import { Button, Typography } from '@mui/material';
+import { FieldArray, FieldArrayRenderProps, useFormikContext } from 'formik';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { ListPagination } from '../../../components/ListPagination';
+import { CristinProject, ProjectFieldName } from '../../../types/project.types';
+import { ROWS_PER_PAGE_OPTIONS } from '../../../utils/constants';
+import { dataTestId } from '../../../utils/dataTestIds';
+import {
+  getNonProjectManagerContributors,
+  hasUnidentifiedContributor,
+} from '../../project/helpers/projectContributorHelpers';
+import { findProjectManagerRole, replaceRolesOnContributor } from '../../project/helpers/projectRoleHelpers';
+import { AddProjectContributorModal } from '../AddProjectContributorModal';
+import { ContributorRow } from './ContributorRow';
+import { ProjectContributorTable } from './ProjectContributorTable';
+
+export const ProjectParticipants = () => {
+  const { t } = useTranslation();
+  const [rowsPerPage, setRowsPerPage] = useState(ROWS_PER_PAGE_OPTIONS[0]);
+  const [currentPage, setCurrentPage] = useState(1);
+  const [openAddProjectParticipantView, setOpenAddProjectParticipantView] = useState(false);
+  const { values, setFieldValue } = useFormikContext<CristinProject>();
+  const contributorsWithNonProjectManagerRole = getNonProjectManagerContributors(values.contributors);
+  const paginatedContributors = contributorsWithNonProjectManagerRole.slice(
+    rowsPerPage * (currentPage - 1),
+    rowsPerPage * currentPage
+  );
+
+  const toggleOpenAddProjectParticipantView = () => setOpenAddProjectParticipantView(!openAddProjectParticipantView);
+
+  const removeProjectParticipant = (name: string, remove: (index: number) => any, contributorIndex: number) => {
+    const projectManagerRole = findProjectManagerRole(values.contributors[contributorIndex]);
+
+    // Contributor also has Project manager role: Only delete all the others
+    if (projectManagerRole) {
+      const newContributors = replaceRolesOnContributor(values.contributors, contributorIndex, [projectManagerRole]);
+      setFieldValue(name, newContributors);
+    } else {
+      // Does not have project manager role: Delete the whole contributor
+      remove(contributorIndex);
+    }
+  };
+
+  return (
+    <>
+      <Typography variant="h2">{t('project.project_participants')}</Typography>
+      <FieldArray name={ProjectFieldName.Contributors}>
+        {({ name, remove }: FieldArrayRenderProps) => (
+          <>
+            <Button
+              sx={{ borderRadius: '1rem', width: '17rem' }}
```

Comment:
antar 17rem ble satt bare for at det skal passe til tekstene som er der i dag?
```suggestion
              sx={{ borderRadius: '1rem', width: 'fit-content' }}
```